plotly/tests/test_optional/test_grid/test_grid.py

```diff
@@ -0,0 +1,32 @@
+"""
+test_grid:
+==========
+
+A module intended for use with Nose.
+
+"""
+from __future__ import absolute_import
+
+from unittest import TestCase
+
+from plotly.exceptions import InputError
+from plotly.grid_objs import Grid
+
+import pandas as pd
+
+
+class Test_Dataframe_to_Grid(TestCase):
```

Comment:
Two things:
1) What does 🐍 mean?
2) Why would DRY be relevant here? What am I repeating?

I'm changing it to your suggested class name, just curious